tests/test_file_download.py

```diff
@@ -51,6 +52,7 @@
 DATASET_SAMPLE_PY_FILE = "custom_squad.py"
 
 
+@with_production_testing
```

Comment:
This decorator enables running test on the hf.co endpoint rather than the moon-staging endpoint. I think it's good practice to explicitly opt-in for production testing